src/lib.rs

```diff
@@ -539,6 +539,27 @@ impl<T: Clone + Num> Mul<Complex<T>> for Complex<T> {
     }
 }
 
+
+// (a + i b) - (c + i d) == (a - c) + i (b - d)
```

Comment:
Please write the correct expansion here, for `(a + i b) * (c + i d) + (e + i f)`. You can show multiple steps, e.g. first fully expanded, then factored such that it's clear where inner `mul_add` can be applied.